packages/column/Column.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import cx from "classnames"
+import styles from "./Column.scss"
+
+export type ColumnProps = {
+  small?: "1" | "2"
+  medium?: "1" | "2" | "3"
+  large?: "1" | "2" | "3"
+  xl?: "1" | "2" | "3" | "4"
+  children?: React.ReactChild
+}
+
+export default function Column({ small, medium, large, xlarge, children }) {
```

Comment:
Was at first wondering, if this should be called `Row` instead (as the elements inside rather are the "columns"), but then I thought this could contain 12+ elements and it would give you a grid kinda layout with multiple rows - a grid rather, wouldn't it? So, in the end - I don't know, but might be worth putting out there.